modules/ROOT/pages/introduction/intro-config-use-fd.adoc

```diff
@@ -0,0 +1,165 @@
+. Click *HTTP Listener*.
+. In the *Path* field, specify the path on which `HTTP Listener` listens for requests.
+. Optionally, use the default CloudHub connection, use another existing connection, or configure a new one, as described in the following sections.
+
+==== Use the Default CloudHub Connection
+
+The default CloudHub connection uses host 0.0.0.0 and port 8081. To use this connection, click outside of the trigger card to close it after you enter the path.
+
+==== Select Another Existing Connection
+
+To select an existing connection other than the default CloudHub connection:
+
+. Click *Edit* on the HTTP Listener trigger card.
++
+.Clicking *Edit* takes you to a screen where you can configure the HTTP Listener connection.
+image::introduction/intro-config-fd-exit.png[CLick Edit]
++
+. Select the connection.
+. Click *Save*.
+
+==== Configure a New Connection
+
+To configure a new connection:
+
+. Click *Edit* on the HTTP Listener trigger card.
+. Click *Add Connection*.
+. In the *Connection Name* field, enter a connection name.
+. In the *Connection Type* field, select *Listener*.
+. Enter values for the *Host* and *Port* fields.
++
+. Optionally configure a reconnection strategy for the new connection:
+.. Scroll down and click *Advanced*.
+.. Optionally enable the *Fails Deployment* field to force the deployment to fail if the connection fails and configured reconnection attempts are exhausted.
+.. In the *Reconnection Strategy* field, select a reconnection strategy.
+.. Complete the fields in the *Reconnection Strategy* section.
++
+For example, this reconnection strategy does the following:
++
+* Forces the deployment to fail if the connection fails and configured reconnection attempts are exhausted
+* Limits the number of reconnection attempts to three at a frequency of every 5000 milliseconds (ms)
++
+.Use the fields in the *Reconnection* section to specify reconnection options.
+image::introduction/intro-config-fd-reconnection-strategy.png[Reconnection strategy example using Flow Designer]
+.. To use TLS, scroll down and specify the required information in the *TLS Configuration* section.
+.. Click *Save*.
+
+For more information about configuring `HTTP Listener`, see the xref:http/http-documentation.adoc[HTTP Connector Reference].
+
+[[scheduler]]
+=== Use Scheduler as a Trigger
+
+Use `Scheduler` to initiate a flow when a time-based condition is met. You can configure `Scheduler` to initiate a flow at a regular interval, or you can configure `Scheduler` to use a more flexible cron expression. A cron expression is a standard for specifying date and time information that initiates a specified action.
+
+To use `Scheduler` as a trigger:
+
+. Follow the procedure described in <<configure-trigger,Configure the Trigger For the Flow>>.
+. In the *Search* field, enter `Scheduler`.
+. Select *Scheduler*.
+. In the *Scheduling Strategy* field, select whether you want to use a fixed frequency or cron expression to trigger the flow:
```

Comment:
ditto re. using caution when inserting examples of task steps into a task

Reply:
Ah! I now see that these are examples of the final step, which is not so worrisome as in the middle of sets of steps.
You could even move these to follow the task, as examples of the final step, if you wanted.